lib/helpers.js

```diff
@@ -60,6 +60,24 @@ exports.checkPropertyName = function(name) {
 };
 
 /**
+ * Validate remote method name. Allows `.` in method name for prototype
+ * @param {String} name The user input
+ * @returns {String|Boolean}
+ */
+exports.validateRemoteMethodName = function(name) {
+  if (name.match(/\./) && !name.match(/^prototype\.(.*)$/)) {
```

Comment:
ah, sorry. I misunderstood. Thought you said it doesn't allow `prototype.`, but you meant any other `.` after `prototype.`. Fixed.